prototype_1/Turbidity_Testing.ino

```diff
@@ -0,0 +1,33 @@
+/*
+ * Example code for DFRobot Turbidity Sensor to test
+*/
+
+void setup() {
+  Serial.begin(9600); //Baud rate: 9600
+}
+void loop() {
+  float average;
+  float readings[10];
+  float NTU;
+  
+  for(int i = 0; i<10; i++) {
+    readings[i] = readTurbidity();
+    delay(200);
```

Comment:
Remember that programming question I asked when you applied... about checking time instead of delaying...